### PR TITLE
Change PhishTank to hard

### PR DESCRIPTION
**Pull request by Wolftallemo**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -8918,8 +8918,9 @@
 
     {
         "name": "PhishTank",
-        "url": "https://www.phishtank.com",
-        "difficulty": "impossible",
+        "url": "https://privacyrequest.cisco.com",
+        "difficulty": "hard",
+        "notes": "Select \"Phishtank\" under \"Account(s) or Service(s) of interest\"",
```
Comment (tupaschoal) [inline _data/sites.json:8923]: Then what? Is there a dropdown stating that the reason is deletion of the account?

Comment (Wolftallemo) [reply _data/sites.json]: That goes in request details (added to note)

Comment (tupaschoal) [reply _data/sites.json]: Thanks

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**